heltour/tournament/templates/tournament/lone_league_home.html

```diff
@@ -64,7 +64,7 @@ <h3>{{ league.name }} TV</h3>
 		                		</a>
 	                		</div>
 		                </div>
-		                <div class="chessground small {% if league.theme == 'green' %}green{% else %}blue{% endif %} cburnett">
+                        <div class="chessground small {% if league.theme == 'green' %}green{% elif league.theme == 'blue' %}blue{% else %}yellow{% endif %} cburnett">
```

Comment:
This class references
https://github.com/cyanfish/heltour/blob/master/heltour/tournament/static/lib/css/cg_theme.css

You could either add a new `yellow` class with a new bg image, or keep it simple and use the `green` class for now (or another one that already exists if you prefer).